spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/elasticsearch/ElasticsearchRestClientProperties.java

```diff
@@ -97,4 +105,19 @@ public void setReadTimeout(Duration readTimeout) {
 		this.readTimeout = readTimeout;
 	}
 
+	public Duration getSniffInterval() {
+		return sniffInterval;
```

Comment:
missing `this.`. This sort of things can be easily spot if you ran the build locally before pushing your changes.